ee/clickhouse/queries/trends/person.py

```diff
@@ -37,19 +37,26 @@ class ClickhouseTrendsActors(ActorBaseQuery):
     entity: Entity
     _filter: Filter
 
-    def __init__(self, team: Team, entity: Optional[Entity], filter: Filter, **kwargs):
+    def __init__(
+        self, team: Team, entity: Optional[Entity], filter: Filter, include_recordings: bool = False, **kwargs
+    ):
         if not entity:
             raise ValueError("Entity is required")
 
         if filter.display != TRENDS_CUMULATIVE and not filter.display in TRENDS_DISPLAY_BY_VALUE:
             filter = _handle_date_interval(filter)
 
+        self._include_recordings = include_recordings
         super().__init__(team, filter, entity, **kwargs)
 
     @cached_property
     def is_aggregating_by_groups(self) -> bool:
         return self.entity.math == "unique_group"
 
+    @cached_property
+    def should_include_recordings(self) -> bool:
```

Comment:
lot of overlapping initialization going on here. Do we expect this property to be set in different ways? If it's just a filter property that will always be read directly maybe should just parse it directly in ActorBaseQuery

Reply:
Good call here. This was a work around because we can't rely on initialization to run in `ActorBaseQuery` because it and some of our other classes are used in multiple-inheritance situations, but aren't setup for that. To get them to work with multi-inheritance is a bit of a refactor, and is going to touch a bunch of files, so I didn't want to add it to this PR.

With that said, what I had was definitely confusing. I cleaned it up and added a comment explaining what was going on.